doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3823,6 +3823,19 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-5-0
+
+NCSDK-24804: TF-M does not withouth gpio0 node enabled in devicetree
+  This fails with error: 'TFM_PERIPHERAL_GPIO0_PIN_MASK_SECURE' undeclared.
+
+  **Workaround:** Enable gpio0 node in devicetree configuration
```

Comment:
```suggestion
  **Workaround:** Enable the GPIO0 node in the devicetree configuration.
```

Reply:
@mia-ko Looking into how we have done this elsewhere, this should be lower-case and with double-backtick,
`TF-M does not compile when a ``uart1`` node`

Reply:
I guess it makes more sense :P